distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Duration.enso

```diff
@@ -191,24 +192,46 @@ type Duration
     hours : Integer
     hours self = @Builtin_Method "Duration.hours"
 
+    ## UNSTABLE
+       Convert the duration to total milliseconds.
+    total_milliseconds : Integer ! Illegal_Argument_Error
+    total_milliseconds self =
+        Panic.catch ArithmeticException (self.total_milliseconds_builtin) _->
+            Error.throw (Illegal_Argument_Error_Data "The duration is too large to convert it to milliseconds")
```

Comment:
minor nitpick: this function has no arguments (okay, other than `self`), so maybe `Illegal_State_Error`? Because the result is independent of arguments - it depends on the internal _state_ of `self`.

But feel free to leave as-is too, I don't think that's very important. Maybe we should decide on some robust guideline when to use which.